careers/careers/static/css/careers-new.css

```diff
@@ -21,12 +28,19 @@
 }
 
 /* nav */
-
 .mzp-c-menu-category .mzp-c-menu-title {
     font-weight: bold;
 }
 
-/* page elements */
+/* $mq-sm in protocol */
+@media only screen and (min-width: 480px) {
```

Comment:
One more, though less important than the job listings. This is also being overridden by the mobile styling later in the cascade. You can move this down to after the other `.locations-list` rules (line 180ish).